{{cookiecutter.project_slug}}/tox.ini

```diff
@@ -1,24 +1,17 @@
 [tox]
-envlist = py34-coverage
+envlist = py34
 skipsdist = True
 
-[base]
-
 [testenv]
-setenv =
```

Comment:
I see the problem. I took the code from manage.py, btw.
Although, If you want to set the environment variables differently than with the envdir, you could still override it... also, the DATABASE_URL is not so much a dummy variable. (of course, SECRET_KEY could easily be added to settings/test.py)

What bothers me is following:
- DATABASE_URL is generated by drifter, the dev would then have to add it manually to tox.ini
- Every time you add an environment variable to envdir, you would have to remember setting it in tox.ini too

Idk what the best solution would be...